docs/sources/datasources/tempo/_index.md

```diff
@@ -55,30 +55,51 @@ You can also configure settings specific to the Tempo data source:
 
 The **Trace to logs** setting configures the [trace to logs feature]({{< relref "../../explore/trace-integration" >}}) that is available when you integrate Grafana with Tempo.
 
-**To configure trace to logs:**
+There are two ways to configure the trace to logs feature. You can use simplified configuration with default query, or you can configure custom query where you can use a template language to interpolate variables from the trace or span.
```

Comment:
Might not be totally clear what template language means in this case

Reply:
Added a link to the variable docs

Reply:
Looks good